package.json

```diff
@@ -16,7 +16,6 @@
   "main": "build/desktop.js",
   "scripts": {
     "preinstall": "make check-version-parity",
-    "postinstall": "make rebuild-deps",
```

Comment:
Unnecessary as `electron build` rebuilds native deps anyway when building artifacts. (Also this somehow didn't jive well when trying to update deps via yarn - perhaps because yarn re-installs and compiles native modules in parallel? 🤷 ) 

For non-final/dev builds (e.g. `make dev`), the makefile will explicitly invoke `make rebuild-deps` so we're covered on both fronts.